website/src/components/EventSlider/index.tsx

```diff
@@ -107,7 +107,7 @@ const EventSlider: React.FC = () => {
               {checkPastDate(event.date) && event.buttonLink && (
                 <Box mt={2} className={classes.actionLInk}>
                   <Link className={classes.linkText} href={event.buttonLink} target="_blank">
-                    <Box display="flex">
+                    <Box display="flex" alignItems="center">
```

Comment:
can we add these props in some syle class

Reply:
Using Box is to avoid writing extra styles, adding extra styles is not ideal